Player1AI/AI.cs

```diff
@@ -28,6 +28,14 @@ public class AIAnt : AIAntBase {
                 Ant.notifyOtherAnts(set);
             }
 
+            if (Ant.isWarrior()) {
+                foreach (BoardObject bObject in view) {
```

Comment:
Soll ja eine gute Beispiel Implementation sein deswegen hier ein ``for`` loop benutzten